packages/web3-providers/src/Firefly/index.ts

```diff
@@ -1,17 +1,18 @@
 import urlcat from 'urlcat'
+import { EMPTY_LIST } from '@masknet/shared-base'
 import { BASE_URL } from './constants.js'
 import { fetchJSON } from '../entry-helpers.js'
 import type { FireflyBaseAPI } from '../entry-types.js'
 
 export class FireflyAPI implements FireflyBaseAPI.Provider {
     async getLensByTwitterId(twitterHandle?: string): Promise<FireflyBaseAPI.LensAccount[]> {
-        if (!twitterHandle) return []
+        if (!twitterHandle) return EMPTY_LIST
```

Comment:
I think it would be safe to only use immutable `EMPTY_{LIST,OBJECT}` in hooks and component props.

In this case above, we can't be sure if the result of `getLensByTwitterId` will be mutated or not. If we return `EMPTY_LIST`, calling `push()` on it will produce unexpected result since `EMPTY_LIST` has been frozen.


Or we must doing mutating carefully.

Reply:
If we need to compose something from the result, `[...result, ...newItems]` is better than `result.push()`.